sdk/core/Azure.Core/tests/TestFramework/RecordEntry.cs

```diff
@@ -75,11 +76,11 @@ private static byte[] DeserializeBody(IDictionary<string, string[]> headers, in
             {
                 if (property.ValueKind == JsonValueKind.Object)
                 {
-                    var arrayBufferWriter = new ArrayBufferWriter<byte>();
-                    using var writer = new Utf8JsonWriter(arrayBufferWriter);
+                    using var memoryStream = new MemoryStream();
```

Comment:
Don't need to have IVT from lib to tests or manually include ArrayBufferWriter into test project anymore.